apps/desktop/desktop/src/main/java/bisq/desktop/main/content/chat/message_container/list/message_box/BubbleMessageBox.java

```diff
@@ -360,4 +359,31 @@ protected static void onCopyMessage(ChatMessage chatMessage) {
     protected static void onCopyMessage(String chatMessageText) {
         ClipboardUtil.copyToClipboard(chatMessageText);
     }
+
+    private void setUpMessageHighlight() {
+        DropShadow dropShadow = new DropShadow();
+        dropShadow.setColor(Color.rgb(86, 174, 72)); // Bisq2 green
+        dropShadow.setBlurType(BlurType.GAUSSIAN);
+
+        Timeline timeline = new Timeline(
+                new KeyFrame(Duration.seconds(10),
+                        new KeyValue(dropShadow.radiusProperty(), dropShadow.getRadius()),
+                        new KeyValue(dropShadow.spreadProperty(), dropShadow.getSpread())),
+                new KeyFrame(Duration.seconds(11),
+                        new KeyValue(dropShadow.radiusProperty(), 0),
+                        new KeyValue(dropShadow.spreadProperty(), 0)));
+        timeline.setOnFinished(e -> messageBgHBox.setEffect(null));
+
+        showHighlightedPin = EasyBind.subscribe(item.getShowHighlighted(), showHighlighted -> {
+            if (showHighlighted) {
+                dropShadow.setRadius(15);
+                dropShadow.setSpread(0.3);
+                messageBgHBox.setEffect(dropShadow);
+                timeline.play();
+            } else {
+                timeline.stop();
+                messageBgHBox.setEffect(null);
```

Comment:
Is that also applied at deactivate? If user switch screen while timeline is running and come back later we need to ensure there is no effect.

Reply:
Looked into src code more closely:
```
timeline.stop();
messageBgHBox.setEffect(null);
```
need to be added to the `dispose` method. timeline needs to be a fields and null checks are required in dispose....